en/docs/index.md

```diff
@@ -356,7 +356,7 @@ template: templates/single-column.html
         <div class="leftContent">
             </br>
             <p>
-                WSO2 API Manager is a complete platform for building, integrating, and exposing your digital services as managed APIs in the cloud, on-premise, and hybrid architectures to drive your digital transformation strategy. 
+                WSO2 API Manager is a complete platform for building, integrating and exposing your digital services as managed APIs across cloud, on-premises, and hybrid architectures to support digital transformation. 
```

Comment:
Better to keep the previous messaging addressing the end user as `drive your digital transformation strategy`